authentik/lib/config.py

```diff
@@ -180,6 +229,21 @@ def update_from_file(self, path: Path):
                 error=str(exc),
             )
 
+    def update_from_dict(self, update: dict):
+        """Update config from dict"""
+        self.__config.update(update)
+
+    @staticmethod
+    def _set_value_for_key_path(outer, path, value, sep="."):
```

Comment:
I think this is due to the rebase, but `set_path_in_dict` should do the same thing as this

Reply:
I have remove the duplicate.